app/constants/repos/repo-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "properties": {
+        "sidebar": {
+            "properties": {
+                "sections": {
+                    "items": {
+                        "properties": {
+                            "items": {
+                                "items": {
+                                    "properties": {
+                                        "href": {
+                                            "type": "string"
+                                        },
+                                        "label": {
+                                            "type": "string"
+                                        }
+                                    },
+                                    "type": "object"
+                                },
+                                "type": "array"
+                            },
+                            "title": {
+                                "type": "string"
+                            }
+                        },
+                        "type": "object"
+                    },
+                    "type": "array"
+                }
+            },
+            "type": "object"
+        }
+    },
+    "type": "object"
+}
+
```

Comment:
later, we could make this available publicly on a remix route so that it could be referenced in a json document in the "$schema" field more easily